test/rpc_test.py

```diff
@@ -237,6 +237,12 @@ def test_self_add(self):
         self.assertEqual(fut.wait(), torch.ones(2, 2) + 1)
         self.assertEqual(ret, torch.ones(2, 2) + 1)
 
+    @dist_init
```

Comment:
We also need to test that we can pass RRef's as arguments to local sync_rpc, async_rpc and remote calls.

Reply:
yes, will add more tests